### PR TITLE
Update for aragonUI 1.0

### PR DESCRIPTION
**Pull request by chadoh**
This fixes some quick things that caused deprecation warnings or had an ugly UI after the update to aragonUI 1.0

It also updates some extra dependencies

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -7,26 +7,26 @@
     "@aragon/api": "2.0.0-beta.8",
     "@aragon/api-react": "2.0.0-beta.8",
     "@aragon/ui": "1.0.0",
-    "@babel/polyfill": "^7.2.5",
+    "@babel/polyfill": "^7.7.0",
     "@githubprimer/octicons-react": "^8.5.0",
     "axios": "^0.19.0",
-    "codemirror": "^5.47.0",
+    "codemirror": "^6.0.0",
```
Comment (PeterMPhillips) [inline app/package.json]: This needs to be fixed:
```suggestion
    "codemirror": "^5.50.0",
```

**Review by PeterMPhillips — CHANGES_REQUESTED** [review]
Looks good except the codemirror version

**Review by e18r — DISMISSED** [review]
LGTM

**Comment by PeterMPhillips** [comment]
Closed because most of the commits were accidentally merged directly into `master`. For everything else, PR #44 covers it

**Pull request closed without merging**